src/tests/ftest/util/user_utils.py

```diff
@@ -113,6 +114,27 @@ def groupadd(log, hosts, group, force=False, sudo=False):
     return run_remote(log, hosts, command)
 
 
+def groupdel(log, hosts, group, force=False, sudo=False):
+    """Run groupdel remotely.
+
+    Args:
+        log (logger): logger for the messages produced by this method
+        hosts (NodeSet): hosts on which to run the command
+        group (str): the group to delete
+        force (bool, optional): whether to use the force option. Default is False
+        sudo (bool, optional): whether to execute commands with sudo. Default is False
+
+    Returns:
+        CommandResult: groups of command results from the same hosts with the same return status
+    """
+    command = ' '.join(filter(None, [
+        'sudo -n' if sudo else None,
```

Comment:
Don't you think it would make sense to standardize on using `command_as_user()` in situations like this?

Reply:
Done